src/VisualStudio/CSharp/Impl/Options/AutomationObject.cs

```diff
@@ -577,6 +577,12 @@ public string Style_UseImplicitTypeWherePossible
             set { SetXmlOption(CSharpCodeStyleOptions.UseImplicitTypeWherePossible, value); }
         }
 
+        public string Style_UseBracesWherePossible
```

Comment:
They were originally together.  Sometime after I implemented this and before I PR'ed, someone added more options, and this is the result of a merge conflict.  I'll fix this.